remill/Arch/Arch.cpp

```diff
@@ -255,6 +255,21 @@ const Arch *GetTargetArch(llvm::LLVMContext &context) {
   return arch;
 }
 
+const remill::Arch *Arch::GetModuleArch(const llvm::Module &module) {
+  auto triple = llvm::Triple(module.getTargetTriple());
+  std::string arch_name = triple.getArchName().data();
```

Comment:
Ideally I'd like you to use a `switch (triple.getArch()` and use that to select an arch, rather than doing string parrsing. Ditto for OS/environment/etc.

Reply:
Fixed in c7b0e04